library/CHART-TEMPLATE/tests/cl-naming_suffix_test.yaml

```diff
@@ -15,17 +15,17 @@ tests:
 
   - it: name is truncated but suffix included
     set:
-      fullnameOverride: including-full-name-as-it-is-would-way-be-longer-than-limit
+      fullnameOverride: including-full-name-as-it-is-would-be-way-longer-than-limit
     asserts:
       - equal:
           path: metadata.name
-          value: including-full-name-as-it-is-would-way-be-longer-than-job-suffix
+          value: including-full-name-as-it-is-would-be-way-longer-tha-job-suffix
 
   - it: suffix is longer than maximum length supported
     set:
       fullnameOverride: fno
-      jobSuffix: this-job-suffix-is-too-long-so-it-is-not-possible-to-include-at-it-is
+      jobSuffix: this-job-suffix-is-too-long-so-it-is-not-possible-to-include-as-it-is
     asserts:
       - equal:
           path: metadata.name
-          value: this-job-suffix-is-too-long-so-it-is-not-possible-to-include-at
+          value: this-job-suffix-is-too-long-so-it-is-not-possible-to-include-as
```

Comment:
NIT: Could we add tests for prefixes and suffixes with exactly 62 and 63 chars?

My rationale is to avoid other issues that we could have in case we fail again doing the maths. I would also want to test the behavior of the dash in these extreme cases.

Reply:
Sure, thanks for pointing out!

Reply:
I added some tests to cover those cases here: [9926128](https://github.com/newrelic/helm-charts/pull/910/commits/9926128253f4ba1396fe2013284acf80ca0c5432)